galaxy_ng/tests/integration/api/test_container_signing.py

```diff
@@ -16,8 +16,15 @@ def flags(ansible_config):
     return api_client("/api/automation-hub/_ui/v1/feature-flags/")
 
 
+@pytest.mark.parametrize(
```

Comment:
Is there a good way to share this decorator?  It's currently repeated a few times across multiple files.

(I don't see an obvious way)

Reply:
Not that I could track down.